public/js/actions/types.js

```diff
@@ -17,8 +17,8 @@ export type { Source };
   * @typedef {Object} ThunkArgs
   */
 export type ThunkArgs = {
-  dispatch: any,
-  getState: any,
+  dispatch: () => Promise<any>,
+  getState: () => any,
```

Comment:
I started some work on making this more explicit, but it needs some tidying up... 
https://github.com/devtools-html/debugger.html/compare/master...jasonLaster:flow-support2?expand=1

anyways, it should be viewed independently of this PR.